spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+describe UsersController do
+  describe "#index" do
+    it "responds successfully" do
+      get users_path
+      expect(response).to be_successful
+    end
+
+    it "returns a 200 response" do
+      get users_path
+      expect(response).to have_http_status "200"
```

Comment:
これはキッチハイクのテストコードの特徴なのですが、
HTTPステータスコードはsymbolで定義されているものを使ってあげるほうがいいです！
```suggestion
      expect(response).to have_http_status :ok
```
https://relishapp.com/rspec/rspec-rails/docs/matchers/have-http-status-matcher